src/main/java/slimeknights/tconstruct/tools/common/network/BouncedPacket.java

```diff
@@ -19,7 +21,8 @@ public void handleClientSafe(NetHandlerPlayClient netHandler) {
 
   @Override
   public void handleServerSafe(NetHandlerPlayServer netHandler) {
-    netHandler.player.fallDistance = 0;
+	if(netHandler.player.getItemStackFromSlot(EntityEquipmentSlot.FEET).getItem() instanceof ItemSlimeBoots)
```

Comment:
Your indentation is way off, we use 2 space indentation. We also always use curly brackets for single if statements

Reply:
Sorry, didn't know. Should be okay now.